src/header/TransferBench.hpp

```diff
@@ -1155,6 +1238,19 @@ namespace {
     hsa_amd_sdma_engine_id_t   sdmaEngineId;      ///< DMA engine ID
 #endif
 
+#ifndef NO_IBV_EXEC
```

Comment:
For consistency, you can add the comment that this section is
// For IBV executor